### DISCUSSION TITLE
Github Pages with Jekyll works on localhost but not on the server

### DISCUSSION THREAD

**Discussion by micxael** [community/community#35131] [Other Feature Feedback, Questions, & Ideas]
Hi everyone!

I’m a newbie who is trying to make a blog with github pages.

I followed a few tutorials and have made a site, and it works locally but when I host it on Github Pages, I don't see any updates changed from the server side. I also have tried rebuilding and redeploying from the pages setting.

Repo - https://github.com/jaeparkim/jaeparkim.github.io
Page - https://jaeparkim.github.io/

This is what it looks like from my local machine.

![Screenshot from 2022-10-04 16-43-53](https://user-images.githubusercontent.com/45185558/193950291-3e8b2971-a763-4540-9bbd-6d589657e55e.png)
 while from  https://jaeparkim.github.io/ is not what I want..

Thank you for your time and kind help!

**Comment by JavierPorron** [comment]
Hi! For me its happening the same... how did you solve it?
